python/perf-kernels/layernorm.py

```diff
@@ -167,7 +174,10 @@ def test_layernorm(M, N, eps=1e-5):
 
 def run_benchmark(args):
     config = []
-    if (args.M_benchmark):
+    sweep_m = args.M_step != 0
+    sweep_n = args.N_step != 0
+    x_vals_list = []
+    if (sweep_m):
```

Comment:
[Non-blocking] Do we need `(...)` in Python `if` statements? Is it a coding pattern for Booleans? Being naive I would just do `if sweep_m:`.

The same suggestions cab be applied to line 190.